workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx

```diff
@@ -0,0 +1,64 @@
+import ClayIcon from '@clayui/icon';
+
+import appIconSales from '../../assets/icons/app-icon-sales.svg';
+import {AppCard} from '../../components/Card/AppCard';
+import {Header} from '../../components/Header/Header';
+
+import './NextStepPage.scss';
+import {Footer} from '../../components/Footer/Footer';
+import {NewAppPageFooterButtons} from '../../components/NewAppPageFooterButtons/NewAppPageFooterButtons';
+
+interface NextStepPageProps {}
+
+export function NextStepPage({}: NextStepPageProps) {
+	return (
+		<>
+			<div className="next-step-page-container">
+				<div className="next-step-page-content">
+					<div className="next-step-page-cards">
+						<AppCard
+							category="Applications"
+							logo={appIconSales}
+							title="Azure Payment"
+						></AppCard>
+
+						<ClayIcon
+							className="next-step-page-icon"
+							symbol="arrow-right-full"
+						/>
+
+						<AppCard
+							category="Applications"
+							logo={appIconSales}
+							title="Azure Payment"
+						></AppCard>
+					</div>
+
+					<div className="next-step-page-text">
+						<Header
+							description="Congratulations on the purchase of Azure Payment. You will now need to configure the app in the Cloud Console. To access the Cloud Console, click the button below and provide your Order ID when prompted."
```

Comment:
@igor-franca instead of displaying "Azure Payment" we need to pass the app name as a parameter to the string. You can consider that the app name will be passed to this page by the parent component.

Reply:
Okay, I realized that I need to create other attributes besides this one to be passed to this page by the parent component. Thanks for the advice!